packages/base/room.gts

```diff
@@ -265,6 +266,19 @@ class MessageCard extends FieldDef {
   static edit = class Edit extends JSONView {};
 }
 
+class AnyField extends FieldDef {
+  static [primitive]: any;
```

Comment:
this is really easily abused. is there any additional structure that you can imbue this with? based on how it is consumed it looks like it always has a `command` property.  probably it always has a payload too?

Reply:
Yeah there's more structure I want to put in this, this was checking if any works as a fundamental type - apart from adding a command type was going to try and make a full json type. Have just realized how to avoid creating cards for the patches at all I think then I can type it much more easily.

Reply:
Structured added for patch commands, and a recursive json type for arbitrary attributes being overridden